charts/v2.0/cray-power-control/values.yaml

```diff
@@ -7,8 +7,8 @@
 #   tag: "" (default = "latest")
 #   pullPolicy: "" (default = "IfNotPresent")
 global:
-  appVersion: 2.0.0
-  testVersion: 2.0.0
+  appVersion: 2.0.1
+  testVersion: 2.0.1
```

Comment:
Change appVersion and testVersion to 2.1.0

Reply:
Fixed